### ISSUE TITLE
Numbered lists could be bold

### ISSUE THREAD

**Issue by Nebucatnetzer** [manubot/rootstock#292]
I wandered if the numbers in a numbered list maybe should be bold.
If we look at https://manubot.github.io/rootstock/#basic-formatting it feels to me that the bullets have more "weight" than the numbers even though they are basically the same item.

**Comment by dhimmel** [comment]
1. Testing
2. What GitHub does

- bullets
- list

Looks like GitHub is also light on the numbered list weight.

Perhaps @vincerubinetti would be able to suggest the CSS to increase the number weight in a numbered list. Then you could evaluate it in your manuscript. For changes like this, we usually like to observe then in the wild a bit before adding them to rootstock.

I don't envision many downsides, perhaps only with how it would interact with numbered lists in bold spans or paragraphs.

**Comment by Nebucatnetzer** [comment]
Maybe it would be better to make the bullets a bit smaller?

Dec. 23, 2019 14:53:26 Daniel Himmelstein <notifications@github.com>:

**Comment by vincerubinetti** [comment]
It is [possible to do this](https://stackoverflow.com/questions/21369843/is-there-any-way-to-make-numbers-in-ordered-list-bold) -- both the number boldening and the bullet shrinking -- but it requires overriding a lot of browser default styles. For such a small feature, it will add many lines of css to maintain.

I'd rather hold off on this until we get more requests for it. If you want this feature, 👍 the original post here.

**Comment by dhimmel** [comment]
> I'd rather hold off on this until we get more requests for it.

Makes sense to me. It seems like we're dealing with pretty accepted default styling of HTML, that perhaps while not ideal, is probably not worth overriding and maintaining.
